serverless/main.mk

```diff
@@ -1,26 +1,39 @@
 # Macroses
 ########################################################################################################################
+PROJ_PATH ?= $(PROJ_DIR)/$(SLS_SERVICE)-$(SLS_FUNCTION)
+
+SLS_DOCKER_IMAGE ?= amaysim/serverless
 SLS_VERSION ?= 1.73.1
-SLS_FILE ?= serverless.yml
+EVENT_FILE ?= event.json
 NODE_VERSION ?= 10.22.0-alpine3.9
 
-SLS ?= @$(DOCKER) run --entrypoint=serverless -v $(ROOT_DIR)/$(PROJECT_PATH):/opt/app -v $(HOME)/.aws/:/root/.aws \
-	-i amaysim/serverless:$(SLS_VERSION)
-NPM ?= @$(DOCKER) run --workdir=/app --entrypoint=npm -v $(ROOT_DIR)/$(PROJECT_PATH):/app -i node:$(NODE_VERSION)
+SLS ?= $(DOCKER) run --rm -e SLS_SERVICE=$(SLS_SERVICE) -e SLS_FUNCTION=$(SLS_FUNCTION) -v $(PROJ_PATH):/opt/app -v ~/.aws/:/root/.aws:ro $(SLS_DOCKER_IMAGE):$(SLS_VERSION) serverless
+NPM ?= $(DOCKER) run --rm --workdir=/app -v $(PROJ_PATH):/app node:$(NODE_VERSION) npm
+
+# Serverless CLI Reference
+########################################################################################################################
+CMD_SLS_INSTALL_PLUGINS = $(NPM) install --save-dev
```

Comment:
This change is not backwards-compatible, and will break other SLS deployments. Id' not change the naming if this is not nessesary

Reply:
fixed